code/__defines/inventory_sizes.dm

```diff
@@ -29,12 +13,15 @@
 	The values below are not yet in use.
 */
 
-#define base_storage_cost(w_class) (2**(w_class-1)) //1,2,4,8,16,...
+#define BASE_STORAGE_COST(w_class) (2**(w_class-1)) //1,2,4,8,16,...
+#define REDUCED_STORAGE_COST(w_class) (2**(w_class-2)) //0.5,1,2,4,8,...
```

Comment:
If anyone has a better idea than this `REDUCED_STORAGE_COST` function I'm ears, if it's an issue it can just be removed.